youtube_dl/extractor/bandcamp.py

```diff
@@ -415,3 +408,52 @@ def _real_extract(self, url):
             'episode_id': compat_str(video_id),
             'formats': formats
         }
+
+
+class BandcampUserIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:user'
+    _VALID_URL = r'https?://(?:(?P<subdomain>[^.]+)\.)?bandcamp\.com/?$'
```

Comment:
No trailing `$`, override `suitable`.

Reply:
Fixed in 02d01a12fb9f4ba165d6f295251bf231268f7277 and c8aa5c673701ef6b62151e0664f41665d2c1b48f